tests/TestCase.php

```diff
@@ -35,7 +35,7 @@ protected function setUp()
         $this->origAccountId = Stripe::getAccountId();
 
         // Set up host and credentials for stripe-mock
-        Stripe::$apiBase = "http://localhost:" . MOCK_PORT;
+        Stripe::$apiBase = MOCK_DOMAIN . ":" . MOCK_PORT;
```

Comment:
The constant should not contain the protocol. Can you change this like so:
```suggestion
        Stripe::$apiBase = "http://" . MOCK_DOMAIN . ":" . MOCK_PORT;
```